services/webnn/ort/error_ort.h

```diff
@@ -24,6 +24,31 @@ namespace webnn::ort {
     }                                                                \
   } while (0);
 
+#define GET_STATUS_PTR(ort_func)                                   \
+  [&]() -> ort::ScopedOrtStatusPtr {                               \
+    OrtStatus* status = ort_func;                                  \
+    ort::ScopedOrtStatusPtr status_ptr;                            \
+    *status_ptr.GetAddressOf() = status;                           \
+    if (status_ptr) {                                              \
+      LOG(ERROR) << "[WebNN] Failed to call " << #ort_func << ": " \
+                 << ort::GetOrtApi()->GetErrorMessage(status_ptr); \
+    }                                                              \
+    return status_ptr;                                             \
+  }()
+
+#define RETURN_STATUS_PTR_IF_FAILED(ort_func)                      \
+  do {                                                             \
+    ort::ScopedOrtStatusPtr status_ptr = GET_STATUS_PTR(ort_func); \
+    if (status_ptr) {                                              \
+      return status_ptr;                                           \
+    }                                                              \
+  } while (0)
+
+#define FAILURE_CAN_BE_IGNORED(ort_func)                           \
```

Comment:
Is this macro needed? Can we just call `GET_STATUS_PTR(ort_func)` or `LOG_IF_FAILED(ort_func)`?

Reply:
Removed.